frontend/src/app/components/canvas/canvas.component.ts

```diff
@@ -396,6 +396,29 @@ export class CanvasComponent implements OnInit, OnDestroy {
     this.canvas.renderAll();
   }
 
+  onResize(event) {
+    let scaleX = event.target.innerWidth / this.canvas.getWidth();
+    // Without toolbar height
+    let scaleY = (event.target.innerHeight - 64) / this.canvas.getHeight();
+    let objects = this.canvas.getObjects();
+
+    // Resize all objects inside the canvas
+    for (var i in objects) {
+      objects[i].scaleX = (objects[i].getObjectScaling().scaleX) * scaleY;
```

Comment:
Is this a typo? Should it be `* scaleX` instead?

Reply:
I made it scale with height only to preserve the post's aspect ratio. This only really matters if the user decides to shrink the window on one axis only.